react/src/pages/utils/components/components.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import {
+    DialogTitle,
+    IconButton,
+    makeStyles,
+    Typography,
+    TypographyProps,
+} from "@material-ui/core";
+import { Close } from "@material-ui/icons";
+import { FieldDisplayValueType } from "../typings";
+
+/**
+ * Returns a simple Typography JSX element for displaying "title: value".
+ */
+export function FieldDisplay(
+    props: TypographyProps & { title: string; value?: FieldDisplayValueType }
+) {
+    let val = props.value;
+    if (Array.isArray(props.value)) val = props.value.join(", ");
+    else if (props.value === null || props.value === undefined) val = "";
+    else if (typeof props.value === "boolean") val = props.value ? "Yes" : "No";
+
+    return (
+        <Typography variant={props.variant ? props.variant : "body1"} gutterBottom>
```

Comment:
If we accept all TypographyProps then we should be able to do `{...props}` at the end of the tag to pass them all down.